README.md

```diff
@@ -15,6 +15,12 @@ System](https://design-system.service.gov.uk/).
 GOV.UK Frontend is maintained by a team at Government Digital Service. If you want to know more about GOV.UK Frontend, please email the [Design System
 team](mailto:govuk-design-system-support@digital.cabinet-office.gov.uk) or get in touch with them on [Slack](https://ukgovernmentdigital.slack.com/messages/govuk-design-system).
 
+### Report a security vulnerability
+
+GDS is an advocate of responsible vulnerability disclosure. If you’ve found a vulnerability, we would like to know so we can fix it.
+
+For full details on how to tell us about vulnerabilities, [see our security policy](https://github.com/alphagov/govuk-design-system/security/policy).
```

Comment:
I think it probably makes sense to link to the security policy associated with this repo (I appreciate it's exactly the same content)

```suggestion
For full details on how to tell us about vulnerabilities, [see our security policy](https://github.com/alphagov/govuk-frontend/security/policy).
```

Reply:
@36degrees Thanks - I'll check the sibling PRs to this one, to see if we also need to link their READMEs to the appropriate repos.

Reply:
I didn't do it in this commit, but I've actioned this feedback. :)